content/english/linux-basics/_index.md

```diff
@@ -1,23 +1,29 @@
 ---
 title: "Linux Basics"
 description: "An introductory guide to using the Linux command line."
-date: 2020-07-08
-prereq: "None"
-difficulty: "Beginner"
+date: 2020-09-17
+prereq: "None."
+difficulty: "Intermediate"
 download: ""
 draft: true
 icon: "fas fa-microchip"
 weight: 1
-translationKey: "linux-basics-main"
 ---
 
 # Welcome
 
-*Greetings! I am Nuvi. We appreciate your coming to assist us!*  
-*We have been notified that our computer is acting strangely and need your help to find out what is going on.*
+*Thank goodness you're here! I am Nuvi, an agent from the Secret Hub Headquarters (SHH). We just got word that someone might have hacked my computer. We need your help to find out what is going on. We're afraid that our secret message might have been leaked!*
 
 ![nuvi picture](images/nuvi.PNG?classes=border,shadow)
 
+An Operating System (OS) is a software which acts as an interface between you and the computer hardware. Every computer must have at least one OS to run other programs. The OS helps you to communicate with the computer without knowing how to speak the computer's language.
+
+Windows is the OS you are probably most familiar with using at home. There are many options out there, but today we will using a Linux based OS, called Kali Linux. It is a popular OS for security experts and hackers!
```

Comment:
Since we expect some people to use Mac OS as well, perhaps mention this briefly too.

Reply:
Added details to include Mac OS.